client/src/app/shared/components/applause-display/applause-display.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="bar-wrapper" *ngIf="isApplauseTypeBar">
```

Comment:
This component is only displaying the applause bar? Please rename it to applause-bar-display or so. Maybe match the name with the particles component, so both have a simslar name.

Reply:
previously, this component was showing both the bar and the particles. That was until Martin and Emanuel issued that the particles should be shown on various entirely different places.
So the "bar" itself is arbitrary, it takes a number and shows how "full" it is without any more logic
the applause display component renders the bar, takes the information from the service and fills the bar with information
the particles component translates the applause level from the service into some amount of unordered "particle cloud stuff".